docs/METRICS.md

```diff
@@ -38,9 +38,9 @@ gauge, or a histogram.
 ## Name Prefixing
 
 To avoid naming conflicts between Proxy-Wasm filters, the name of a metric is
-always prefixed with: `pw.{filter_name}.{metric_name}`. This means that a metric
-named `a_counter` inserted by `a_filter` will have its name stored as:
-`pw.a_filter.a_counter`.
+always prefixed with space-separated metadata: `pw {filter_name} `. This means
```

Comment:
Reading this I realized space might not be the ideal character; it's easy to miss the trailing space in docs describing `pw {filter_name} `. Would it be fine if we used colons like `pw:{filter_name}:`? I opened #619 for merge if that's alright.

Reply:
Indeed, colons are more easily spotted than spaces.